packages/types/src/index.ts

```diff
@@ -13,7 +13,6 @@ export { Options } from './options';
 export { Package } from './package';
 export { Request } from './request';
 export { Response } from './response';
-export { Runtime } from './runtime';
```

Comment:
I'm not sure about this, I thought @kamilogorek meant to add a new file.

`Runtime` is imported in `packages/node/test/handlers.test.ts`.